.travis.yml

```diff
@@ -7,9 +7,16 @@ python:
   - "pypy"
 
 install:
-  - "python setup.py --version"
-  - "travis_retry pip install -r requirements.txt"
-  - echo ${TRAVIS_PYTHON_VERSION}
+  - travis_retry pip install -r requirements.txt
+  - travis_retry pip install python-coveralls coverage nose-cov
+  - echo TRAVIS_PYTHON_VERSION=${TRAVIS_PYTHON_VERSION}
   - if [ ${TRAVIS_PYTHON_VERSION:0:1} != "3" ]; then travis_retry pip install contextlib2; fi
+  - python setup.py --version
 
-script: nosetests
+# Run test
+script:
+  - nosetests -v --with-cov --cov streamparse --cov-config .coveragerc --logging-level=DEBUG
+
+# Calculate coverage on success
+after_success:
+  - coveralls --config_file .coveragerc
```

Comment:
My vote is either 2.7 or 3.4.  If we want to live in the future, we can just do 3.4.  The coverage information ought to be the same for them all anyway, no?